packages/components/src/Gallery/Gallery.test.tsx

```diff
@@ -167,3 +172,33 @@ describe("when a non-image is clicked", () => {
     });
   });
 });
+
+describe("Thumbnails", () => {
+  it.each(files.map(file => [file.name, file.thumbnailSrc]))(
+    "should display the thumbnailSrc as thumbnails for %s",
```

Comment:
Super small nitpick, I was confused when I read the test title, do you think this might be a little clearer? 

"should use the thumbnailSrc as the image source for %s",

Reply:
that makes sense